lib/bitcoin/electrum_wallet.dart

```diff
@@ -174,28 +172,75 @@ abstract class ElectrumWalletBase extends WalletBase<ElectrumBalance,
       }
     }
 
-    if (inputs.isEmpty ||
-        (credentialsAmount > 0 && allInputsAmount < credentialsAmount)) {
+    if (inputs.isEmpty) {
       throw BitcoinTransactionNoInputsException();
     }
 
-    final allAmountFee =
-      feeAmountForPriority(transactionCredentials.priority, inputs.length, 1);
+    final allAmountFee = feeAmountForPriority(
+        transactionCredentials.priority, inputs.length, sendItemList.length);
     final allAmount = allInputsAmount - allAmountFee;
 
-    final amount = transactionCredentials.amount == null ||
-        allAmount - credentialsAmount < minAmount
-        ? allAmount
-        : credentialsAmount;
-    final fee = transactionCredentials.amount == null || amount == allAmount
-        ? allAmountFee
-        : calculateEstimatedFee(transactionCredentials.priority, amount);
+    var credentialsAmount = 0;
+    var amount = 0;
+    var fee = 0;
+
+    if (sendItemList.length > 1) {
```

Comment:
better to hold result of that condition into "hasMultiDestination" variable, because then you could to use it into next logic blocks